src/app/components/toolbar/index.html

```diff
@@ -20,6 +20,13 @@
     >
     Specifies a section to group items within the toolbar. The section displays items in a separate horizontal row.
     </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="sky-toolbar-view-actions"
+      isOptional="true"
+    >
+    Adds a section on the right side of the toolbar for items that substantially change or affect the view of the toolbar. This includes simple filters and view switchers. If the view section includes more than one item, simple filters appear on the left and view switchers appear on the right.
```

Comment:
`Adds a section on the right side of the toolbar for items that alter the view of the toolbar.` is not really true. These items will affect the "view below". The usage guideline state that the items in this section are:
`items that substantially change or affect the view of the rest of the content container.`